logfire/_internal/config.py

```diff
@@ -1228,3 +1238,7 @@ def sanitize_project_name(name: str) -> str:
 
 def default_project_name():
     return sanitize_project_name(os.path.basename(os.getcwd()))
+
+
+class LogfireNotConfiguredWarning(UserWarning):
```

Comment:
should this be public?

Reply:
Currently we have a few warnings but none are public. I think that if we decide to make them public we should make a new `logfire.warnings` module for them in a new PR.